src/main.rs

```diff
@@ -55,24 +50,21 @@ fn main() -> std::io::Result<()> {
         .expect("Can't create runtime.");
     let runtime_handle = runtime.handle().clone();
 
-    #[allow(unused_mut)]
-    let mut toc = TableOfContent::new(&settings.storage, runtime);
+    let mut toc = TableOfContent::new(&settings.storage, runtime, settings.cluster.enabled);
     runtime_handle.block_on(async {
         for collection in toc.all_collections().await {
             log::info!("Loaded collection: {}", collection);
         }
     });
 
-    #[cfg(feature = "consensus")]
     let (propose_sender, propose_receiver) = std::sync::mpsc::channel();
```

Comment:
the `channel` and the `propose_sender` are always initialized regardless of the configuration,